### PR TITLE
Update basic_operations.rst

### LINKED ISSUE #11990: lambdify() docs need a "Warning: Uses eval()"
[sympify()](http://docs.sympy.org/latest/tutorial/basic_operations.html#converting-strings-to-sympy-expressions) has a warning: 

> Warning 
> sympify uses eval. Don’t use it on unsanitized input. 

`lambdify()` needs it as well since: 
```

import sympy 
some_str =  'exec("import os;print(os.getcwd())")'
sympy.lambdify(x, some_str, (sympy, numpy)) 

```

PS: How can I PR the doc instead of asking you to do so? I m relatively new to git and PRs.

### PR DESCRIPTION
**Pull request by jtoka**
Fixes #11990

### CODE REVIEW

**Comment by jtoka** [comment]
@moorepants added to docstring as you suggested. Also added to ```sympify```'s docstring. Ready for re-review.

**Review by moorepants — APPROVED** [review]

File: sympy/core/sympify.py [hunk 1]
```diff
@@ -66,6 +66,9 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
        - booleans, including ``None`` (will leave ``None`` unchanged)
        - lists, sets or tuples containing any of the above
 
+    Note that this function uses ``eval``, and thus shouldn't be used on
```
Comment (asmeurer) [inline sympy/core/sympify.py]: These should also use the `.. warning` directive.

Comment (moorepants) [inline sympy/core/sympify.py]: @jtoka or @asmeurer, feel free to fix in another pr. i merged before this comment came in.

Comment (jtoka) [inline sympy/core/sympify.py]: @asmeurer @moorepants  I'm new to Sympy...what's the process for handling this? Do I first create an issue around it to link the PR to or should I just create a PR and reference this issue?

Comment (moorepants) [inline sympy/core/sympify.py]: No need to create an issue. Just submit a quick PR.

Comment (jtoka) [inline sympy/core/sympify.py]: @moorepants whoops beat you to it :). Should I close it?

**Pull request merged**